specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/stable/2016-06-01/examples/DeleteRegisteredIdentities.json

```diff
@@ -0,0 +1,14 @@
+{
+    "parameters":{
+         "subscriptionId": "04cf684a-d41f-4550-9f70-7708a3a2283b",
```

Comment:
Is this a real subscription ID? If it is, we should scrub it.

Reply:
Yes, This is a real subscriptionId, Updating it. Thanks for your suggestion.